sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md

```diff
@@ -0,0 +1,103 @@
+# Analyze a conversation
+
+This sample demonstrates how to analyze a conversation with Conversation Summarization. To get started, you'll need to create a Cognitive Language service endpoint and an API key. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md) for links and instructions.
+
+To analyze an utterance, you need to first create a `ConversationAnalysisClient` using an endpoint and API key. These can be stored in an environment variable, configuration setting, or any way that works for your application.
+
+```C# Snippet:ConversationAnalysisClient_Create
+Uri endpoint = new Uri("https://myaccount.cognitive.microsoft.com");
+AzureKeyCredential credential = new AzureKeyCredential("{api-key}");
+
+ConversationAnalysisClient client = new ConversationAnalysisClient(endpoint, credential);
+```
+
+Once you have created a client, you can call synchronous or asynchronous methods.
+
+## Synchronous
+
+```C# Snippet:AnalyzeConversation_ConversationSummarization
+var data = new
+{
+    analysisInput = new
+    {
+        conversations = new[]
+        {
+            new
+            {
+                conversationItems = new[]
+                {
+                    new
+                    {
+                        text = "Hello, how can I help you?",
+                        id = "1",
+                        participantId = "Agent",
+                    },
+                    new
+                    {
+                        text = "How to upgrade Office? I am getting error messages the whole day.",
+                        id = "2",
+                        participantId = "Customer",
+                    },
+                    new
+                    {
+                        text = "Press the upgrade button please. Then sign in and follow the instructions.",
+                        id = "3",
+                        participantId = "Agent",
+                    },
+                },
+                id = "1",
+                language = "en",
+                modality = "text",
+            },
+        }
+    },
+    tasks = new[]
+    {
+        new
+        {
+            parameters = new
+            {
+                summaryAspects = new[]
+                {
+                    "issue",
+                    "resolution",
+                }
+            },
+            kind = "ConversationalSummarizationTask",
+            taskName = "1",
+        },
+    },
+};
+
+Operation<BinaryData> analyzeConversationOperation = client.SubmitJob(WaitUntil.Started, RequestContent.Create(data));
```

Comment:
I thought it was called AnalyzeConversation?

Reply:
It is. I forgot to run our snippet updater.